extensions/shared/upgrade-nudge/index.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { Button } from '@wordpress/components';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import getSiteFragment from '../get-site-fragment';
+
+import './style.scss';
+
+export default ( { feature, plan } ) => (
+	<div className="upgrade-nudge">
+		<Gridicon className="upgrade-nudge__icon" icon="star" size={ 18 } />
+		<div className="upgrade-nudge__info">
+			<span className="upgrade-nudge__title">
+				{ sprintf( __( 'You need at least the following plan: %(plan)s', 'jetpack' ), { plan } ) }
+			</span>
+			<span className="upgrade-nudge__message">{ __( 'To gain access to this block.' ) }</span>
```

Comment:
This seems like a single phrase that should be localized all together. Maybe the following?

```
To access this block, you need at least the following plan: %(plan)s
```

Reply:
More copy discussion on this thread: paAmJe-ll-p2. Going with 9669ec3cfcdd81e7035d37995cd9c35f2f285ae8 for now.